task-sdk/src/airflow/sdk/bases/xcom.py

```diff
@@ -274,6 +281,61 @@ def get_one(
         )
         return None
 
+    @classmethod
+    def get_all(
+        cls,
+        *,
+        key: str,
+        dag_id: str,
+        task_id: str,
+        run_id: str,
+    ) -> Any | None:
+        """
+        Retrieve all XCom values for a task, typically from all map indexes.
+
+        This method returns "full" XCom values (i.e. uses ``deserialize_value``
+        from the XCom backend).
```

Comment:
But this doesn’t? It never calls `deserialize_value`.

Reply:
Copy paste error :D
removing it

Reply:
Handled in [review comments from TP](https://github.com/apache/airflow/pull/51568/commits/a2c01b1176f41e29dbd5238e93b14a231216441a)